Doc/c-api/intro.rst

```diff
@@ -127,15 +127,15 @@ complete listing.
    item defined in the module file. Example::
 
        static struct PyModuleDef spam_module = {
-           PyModuleDef_HEAD_INIT,
+           .m_base =PyModuleDef_HEAD_INIT,
            .m_name = "spam",
            ...
        };
```

Comment:
Would adding `.m_slots` here distract from the motivating case of `PyMODINIT_FUNC`? If not, may be worth adding it (but omit the slot struct definition)